### DISCUSSION TITLE
Add ShimmerEVM to list of chains

### DISCUSSION THREAD

**Discussion by MarkNerdi** [WalletConnect/walletconnect-monorepo#3933] [Developer Support] [answered]
Hello :wave: 

Our newly released wallet will be supporting `ShimmerEVM` and its testnet (ref: https://explorer.evm.shimmer.network/). Could you add this please to the dropdown in the `Explore` tab on the dashboard :pray: 

`ShimmerEVM`: `eip155:148`
`ShimmerEVM Testnet`: `eip155:1073`

Or is that dashboard open source by chance? If yes, i'd create a PR for that.

Thanks!

**Comment by glitch-txs** [comment, marked as answer]
Hi! Thanks for this! You can request this following these steps https://docs.walletconnect.com/advanced/multichain/overview
